Reqnroll.Generator/UnitTestProvider/MsTestGeneratorProvider.cs

```diff
@@ -28,16 +29,19 @@ public class MsTestGeneratorProvider : IUnitTestGeneratorProvider
 
         protected CodeDomHelper CodeDomHelper { get; set; }
 
+        protected ReqnrollConfiguration ReqnrollConfiguration { get; set; }
+
         public virtual UnitTestGeneratorTraits GetTraits()
         {
             return UnitTestGeneratorTraits.None;
         }
 
         public bool GenerateParallelCodeForFeature { get; set; }
 
-        public MsTestGeneratorProvider(CodeDomHelper codeDomHelper)
+        public MsTestGeneratorProvider(CodeDomHelper codeDomHelper, ReqnrollConfiguration reqnrollConfiguration)
```

Comment:
It seems the `ReqnrollConfiguration` is already [registered and available](https://github.com/reqnroll/Reqnroll/blob/d62fe3330f1245945824a7cda3e457733c8cb168/Reqnroll.Generator/GeneratorContainerBuilder.cs#L42) to generators, so I don't believe any additional logic should be necessary here. But please let me know if that's mistaken!